### DISCUSSION TITLE
直播默认选择原画画质

### DISCUSSION THREAD

**Discussion by CuewarsTaner** [the1812/Bilibili-Evolved#2883] [功能建议 / Ideas]
![123](https://user-images.githubusercontent.com/46869194/149274398-683e0dc7-7d44-41bb-a8e2-4829837a5740.png)
B站直播，每次进入直播间都是默认超清画质，每次都要改清晰度。
请问能否做一个功能插件，让我们进入直播间后自动选择最清晰的解析度？
谢谢！

**Comment by Geeyun-JY3** [comment]
请先善用搜索，一个问这个问题的 discussion 就在你上面：#2881
